pkg/collector/corechecks/embed/jmx/loader.go

```diff
@@ -61,17 +76,18 @@ func (jl *JMXCheckLoader) Load(config integration.Config) ([]check.Check, error)
 	}
 
 	for _, instance := range config.Instances {
-		if err = jmxLauncher.Configure(instance, config.InitConfig); err != nil {
+		if err = state.runner.configureRunner(instance, config.InitConfig); err != nil {
```

Comment:
Now that we can know the state.... Does it make sense to try to configure the runner if the process is already running?